### ISSUE TITLE
StandaloneWindows64

### ISSUE THREAD

**Issue by andreas-rocketbike** [modesttree/Projeny#61]
Hi,

when I switch to a 64 Bit build projeny complains that I switched platforms outside projeny and changes back to StandaloneWindows. Any way to make 64 Bit default?

**Comment by svermeulen** [comment]
Yes, I encountered this same problem with it.  I'll add an option for it.

**Comment by svermeulen** [comment]
Ok so now:
- It should no longer complain when changing between 32 bit and 64 bit
- You can get it to default to 64 bit by including this in your Projeny.yaml file:
```
Unity:
    Win64IsDefault: True
```
I haven't had time to work on this for awhile because I'm waiting until I have enough time to do a major refactor.  I want to port all the python to C#, and also I want to change the way it handles multiple projects.  Right now it assumes one project for each platform, but I've been thinking that it might be better to just allow any number of multiple unity projects per projeny project, with some settings to control what platform each project should be including platform-specific properties.  This would also be much nicer because you could have multiple project instances with the same config.  You might use this to test networking on windows and debug both sides with unity, or just to have the same project open twice for some other reason.

**Comment by andreas-rocketbike** [comment]
Hi thanks for the quick reply. Will check out the stuff soon. Making the platforms more flexible would be really cool as we will have multipme Windows builds (Oculus, Vive) and also multiple Android builds and they all need some different Packages and Settings.

**Comment by svermeulen** [comment]
Exactly yeah - having different per-project configs that generate different unity projects could allow each unity project to have different packages too
